src/skins/vector/css/matrix-react-sdk/structures/_MatrixChat.scss

```diff
@@ -100,6 +100,11 @@ limitations under the License.
      * flex itself.
```

Comment:
I have now and as far as I can tell, it's fine. (Albeit I'm testing with Chrome)